src/main/java/bowling/domain/Pin.java

```diff
@@ -4,61 +4,70 @@
 
 public class Pin {
 
-    private static final int MIN_PIN_COUNT = 0;
-    private static final int MAX_PIN_COUNT = 10;
-    private static final String INVALID_PIN_COUNT = "잘못된 수를 입력했습니다.";
-    private final int count;
+  private static final int MIN_PIN_COUNT = 0;
+  private static final int MAX_PIN_COUNT = 10;
+  private static final String INVALID_PIN_COUNT = "잘못된 수를 입력했습니다.";
+  private final int count;
 
 
-    public Pin(int count) {
-        validate(count);
-        this.count = count;
-    }
+  public Pin(int count) {
+    validate(count);
+    this.count = count;
+  }
 
-    private void validate(int count) {
-        if (count < MIN_PIN_COUNT || count > MAX_PIN_COUNT) {
-            throw new IllegalArgumentException(INVALID_PIN_COUNT);
-        }
+  private void validate(int count) {
+    if (count < MIN_PIN_COUNT || count > MAX_PIN_COUNT) {
+      throw new IllegalArgumentException(INVALID_PIN_COUNT);
     }
+  }
 
-    public boolean isEnd() {
-        return count == MAX_PIN_COUNT;
-    }
+  public boolean isEnd() {
+    return count == MAX_PIN_COUNT;
+  }
 
-    public boolean isSpare(Pin fallenPins) {
-        return this.count + fallenPins.getCount() == MAX_PIN_COUNT;
-    }
+  public boolean isSpare(Pin fallenPins) {
+    return this.count + fallenPins.getCount() == MAX_PIN_COUNT;
```

Comment:
자기 자신이기에 getter말고 바로 접근하시면 좋을 것 같아요 :)
```suggestion
    return this.count + fallenPins.count == MAX_PIN_COUNT;
```